docling/backend/genos_hwp_backend.py

```
@@ -0,0 +1,653 @@
+
+            # 4-a) SDK 실행 명령어 구성
+            cmd = [
+                "convtext", 
+                str(self.source_path.resolve()), 
+                str(json_out.resolve()), 
+                str(info_out.resolve()), 
+                img_path_str
+            ]
+            
+            # 4-b) 실제 SDK 실행
+            subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)